### PR TITLE
[WIP] Documentation Revision 1

### PR DESCRIPTION
**Pull request by evildandelions**
Improving documentation readability, grammar, structure

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/990)

### CODE REVIEW

**Comment by kris-nova** [comment]
@k8s-bot ok to test

**Comment by evildandelions** [comment]
I signed it!

**Comment by kris-nova** [comment]
Bump - @evildandelions can we rebase this please?

File: docs/adding_a_feature.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
-This is an overview of how we added a feature:
+## Adding a Feature
```
Comment (chrislovecnm) [inline docs/adding_a_feature.md]: Should we talk what we are actually adding a feature to?

File: docs/adding_a_feature.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
 
-To make auto-upgrading on the nodes an option.
+###Making auto-upgrading on the nodes an option
```
Comment (chrislovecnm) [inline docs/adding_a_feature.md]: So this documentation is actually an example of adding a feature that exists.  We should probably mention that this is an example.

**Comment by chrislovecnm** [comment]
We need a rebase as well please.

**Comment by kris-nova** [comment]
Closing this - will have @evildandelions reopen a new PR after 1.5

**Pull request closed without merging**